extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthRuntimeConfig.java

```diff
@@ -76,4 +76,51 @@ public interface AuthRuntimeConfig {
      * Form Auth config
      */
     FormAuthRuntimeConfig form();
+
+    /**
+     * Require that all registered HTTP authentication mechanisms must attempt to verify the request credentials.
+     * <p>
+     * By default, when the {@link #inclusiveMode} is strict, every registered authentication mechanism must produce
+     * SecurityIdentity, otherwise, a number of mechanisms which produce the identity may be less than a total
+     * number of registered mechanisms.
+     * <p>
+     * The identities can be retrieved using utility method as in the example below:
+     *
+     * <pre>
```

Comment:
This code fragment is not complete. May be we should only say above that all the security identities can be retrieved using the `io.quarkus.vertx.http.runtime.security.HttpSecurityUtils.getSecurityIdentities` utility method which accepts an injected `SecurityIdentity` ?